module-1/lab-api-scavenger-game/your-code/API_Challenge_3.ipynb

```diff
@@ -0,0 +1,1088 @@
+{
```

Comment:
You were near to complete!! 
If you would had normalized the JSON, it would have easier to get the clean list of ordered files.